torch/utils/data/dataloader.py

```diff
@@ -567,20 +572,26 @@ def _get_shared_seed(self):
                 ws = dist.get_world_size()
                 store = dist.distributed_c10d._get_default_store()
                 if rank == 0:
-                    store.set("_dl_shared_seed", str(_shared_seed))
+                    _shared_seed_str = str(_shared_seed)
+                    store.set("_dl_shared_seed", _shared_seed_str)
+                    logger.info(f"Shared seed ({_shared_seed_str}) sent to store on rank 0")
+                    # Use 'add' instead of 'get' since for some store implementations 'add'
+                    # doesn't work well with 'get'.
+                    _shared_seed_recv_cnt = store.add("_dl_shared_seed_recv_cnt", 1)
+                    while _shared_seed_recv_cnt < ws:
+                        time.sleep(0.01)
```

Comment:
snit: Sleep interval is good candidate to be constant

Reply:
Makes sense.